docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -395,23 +398,50 @@ See Also
 ::::::::
 ListEditor, TextEditor
 
-DefaultOverride()
-`````````````````
+
+DateEditor()
+````````````
 
 :Suitable for:
-    (any)
+    Date, List(Date) (custom style only)
 :Default for:
-    (none)
+    Date
+:Optional parameters:
+    *allow_future*, *message*, *months*, *multi_select*, *on_mixed_select*,
+    *padding*, *shift_to_select*, *selected_style*, *strftime*, *view*
+
+The DateEditor() displays a Python datetime date object, usually supplied via a
+Date trait.  The simple style shows a date spin control, while the custom style
+shows one (or potentially more in the wx backend) months in a calendar view.
+Dates can be restricted to past-only by setting *allow_future* to False.
```

Comment:
I had no idea that `allow_future` was an option. Given that it is, though, it seems strange that `allow_past` is not. But that's off-topic for this PR.